services/index.js

```diff
@@ -1,20 +1,31 @@
 'use strict'
 
 const glob = require('glob')
+const { BaseService } = require('./base')
 
 function loadServiceClasses() {
   // New-style services
-  const services = glob
-    .sync(`${__dirname}/**/*.service.js`)
-    .map(path => require(path))
+  const servicePaths = glob.sync(`${__dirname}/**/*.service.js`)
 
   const serviceClasses = []
-  services.forEach(service => {
-    if (typeof service === 'function') {
-      serviceClasses.push(service)
+  servicePaths.forEach(path => {
+    const module = require(path)
+    if (!module) {
```

Comment:
I don't think this is working as you expect. A module that exports nothing gives `module` the value `{}` here but in that situation this condition is true because `{}` is truthy in javascript. Also if we explicitly `module.exports = []`, `[]` is also truthy, but we would want to follow this path in that situation too.

Reply:
Ahh, yes, good point.